mz_bokeh_package/components/confirmation_modal.py

```diff
@@ -46,29 +46,29 @@ def __init__(self, title: str, content_widgets: List[Widget], state: AppState):
         self._modal_invoker.js_on_change("active", self._get_modal_invoker_js_callback())
 
         # "Apply" button.
-        self._apply_modal_btn = Button(
+        self.apply_modal_btn = Button(
```

Comment:
@roiweinreb I don't think it's a good idea to expose the actual button when all you need to set is its label. Better to create a property (using the `@property` and `setter` decorators), something like `apply_button_label` and `cancel_button_label`.